server/token.ts

```diff
@@ -1,18 +1,23 @@
+import * as fs from "fs";
 import * as process from "process";
 import logger from "./logger";
 import api from "../common/api";
 
+// All secret files in render are available to load by absolute path at /etc/secrets/<filename>.
```

Comment:
Let's also point out

1. How local development use env to pass the token.

2. For bytebase, where are the donated token stored.

Reply:
Updated. PTAL